README.md

```diff
@@ -1,6 +1,32 @@
 # powerranger
 
-lf
+A ranger-inspired file manager for PowerShell.
+
+## Installation
+
+You'll need Python 3.8 or higher. Install or update with:
+
+```shell
+pip install --user --upgrade powerranger
+```
+
+## Usage
+
+This program is intended to run on Windows. From `powershell.exe` or
+`cmd.exe`, you can run:
+
+```cmd
+python -m powerranger
```

Comment:
Is this tested?

I'm not sure `-m` will work without module-level handling.

Reply:
This is untested so far, but probably won't work anyways. The idea is to keep an eye on the future. I'd like a Script to be installed as well as the ability to call with `python -m powerranger`. If `-m` isn't possible, then I'll happily settle for having a `site-packages/Scripts`-only variant, but, again, this is looking ahead only.